components/applications-service/integration_test/service_groups_test.go

```diff
@@ -25,7 +25,7 @@ func TestServiceGroupsBasic(t *testing.T) {
 	assert.Equal(t, expected, response)
 }
 
-func TestServiceGroupsHealthOneOk(t *testing.T) {
+func TestGetServiceGroupsOneOk(t *testing.T) {
```

Comment:
All these test names where confusing since we are testing the rpc function `GetServicweGroups` not the `ServiceGroupsHealthCounts`.